cmd/kops/create_cluster.go

```diff
@@ -292,6 +296,10 @@ func NewCmdCreateCluster(f *util.Factory, out io.Writer) *cobra.Command {
 		cmd.Flags().StringVar(&options.VSphereCoreDNSServer, "vsphere-coredns-server", options.VSphereCoreDNSServer, "vsphere-coredns-server is required for vSphere.")
 		cmd.Flags().StringVar(&options.VSphereDatastore, "vsphere-datastore", options.VSphereDatastore, "vsphere-datastore is required for vSphere.  Set a valid datastore in which to store dynamic provision volumes.")
 	}
+
+	if featureflag.EnableExternalCloudController.Enabled() {
+		cmd.Flags().BoolVar(&options.EnableExternalCloudController, "enable-external-cloud-controller", options.EnableExternalCloudController, "Enable the cloud controller manager introduced in kubernetes v1.7")
```

Comment:
I am apprehensive to add this as another cli option.  Why does a basic user need direct access to this option?  Is this enabling the cloud controller manager?  Need more context please.